testing/internal/e2e/tests/static/connect_authz_token_test.go

```diff
@@ -52,8 +52,8 @@ func TestCliConnectTargetWithAuthzToken(t *testing.T) {
 	require.NoError(t, err)
 
 	newSessionAuthorization := newSessionAuthorizationResult.Item
-	retrievedUser := fmt.Sprintf("%s", newSessionAuthorization.Credentials[0].Credential["username"])
-	retrievedKey := fmt.Sprintf("%s\n", newSessionAuthorization.Credentials[0].Credential["private_key"])
+	retrievedUser := newSessionAuthorization.Credentials[0].Credential["username"].(string)
```

Comment:
FYI these are called `type assertions` and can fail (with a panic) at runtime. I think we're confident of these types in this case, but if you're doing this and you're not _sure_ it's the right type, you can use an `_, ok` pattern to handle the error without panicing.
```go
	retrievedUser, ok := newSessionAuthorization.Credentials[0].Credential["username"].(string)
```

Reply:
To confirm, it is preferred to use a `type assertion` here, but we should do error checking here to prevent panics?

Reply:
Chatted with Johan. We're good to go.